std/node/process.ts

```diff
@@ -0,0 +1,32 @@
+export const version = `v${Deno.version.deno}`;
+
+export const versions = {
+  node: Deno.version.deno,
+  ...Deno.version
+};
+
+const osToPlatform = (os: Deno.OperatingSystem): string =>
+  os === "win" ? "win32" : os === "mac" ? "darwin" : os;
+
+export const platform = osToPlatform(Deno.build.os);
+
+export const { arch } = Deno.build;
+
+export const argv = [Deno.execPath(), ...Deno.args];
+
+// TODO(rsp): currently setting env seems to be working by modifying the object
+// that is returnd by Deno.env(). Need to make sure that this is the final API
+// or Deno.env('key', 'value') is to be used in the future.
+export const env = Deno.env();
```

Comment:
Can you use a getter here? Otherwise everyone who imports this module will need to have --allow-env

Reply:
Ok, so you mean to give only a read access to the env? (I thought that `--allow-env` was needed to read the env as well, wasn't it?)

If so then I can either return a copy of the env as it was in the moment of reading, or use a Proxy to handle getting properties and throw on setting them?

Reply:
Oh, I think I know what you mean :) Never mind what I said, I apparently had too little sleep today.